tardis/__init__.py

```diff
@@ -34,9 +34,9 @@
 logging.getLogger("py.warnings").addHandler(console_handler)
 
 # ----------------------------------------------------------------------------
-# pyne holds Python 3.7 on macOS, but refdata is pickled with protocol 5 (3.8)
+# pyne holds Python 3.7 on macOS, but refdata is pickled with protocol 5 (3.8.3)
 
-if sys.version_info < (3, 8):
+if sys.version_info < (3, 8, 3):
```

Comment:
@epassaro, oh, I think this check is not needed at all. pickle5 is only available on < 3.8 through conda, the whole tardis environment now resolves to Python 3.7 after #1566 because of this (macOS AND Linux).

Reply:
The data is already pickled with 3.8, so yes, we need this

Reply:
I think I'm missing something, can you point at the workflow with Python 3.8, please?